xdsl/dialects/riscv_func.py

```diff
@@ -48,9 +51,96 @@ def verify_(self):
             )
 
 
+@irdl_op_definition
+class CallOp(IRDLOperation):
```

Comment:
Could you please add a doc comment to this operation and below? Ideally short and descriptive

Reply:
Added short doc comments, let me know if they are not descriptive enough